### ISSUE TITLE
UW to confirm: which ISRM was used to calculate the results on p. 20 of Apte et al (2019)?a. the ca_isrm.ncf posted on GDrive (n = 21,705 "California" cells, 2,547 of which are "SFAB" if we cut using the [CMAQ envelope]), orb. the isrm_v1.2.1.zip posted on Zenodo (52,411 "contiguous US" cells, n = 9,001 of which are "California" if that's how we interpret ca_isrm_gridcells.csv, which has 9,001 rows)?

### ISSUE THREAD

**Issue by dholstius** [BAAQMD/InMAP-SFAB#5]

**Comment by dholstius** [comment]
It seems like we should only comparing the "Richmond iF example" (p. 20 of CARB report) to issue #1 if they share the same basis (i.e., ISRM).

**Comment by dholstius** [comment]
I think most of the content we've been talking about for the "demo handoff" has been based on the US ISRM. The Bay Area subset, that is. Should we be talking about / using that at all?

**Comment by joshapte** [comment]
The CARB report you reference (and the Richmond example) uses the 21k cell ISRM matrix we developed for this project.

**Comment by bujinb** [comment]
Yes, the demo was done using the US ISRM which has ~9000 grid cells for California.

**Comment by joshapte** [comment]
I would be surprised if the two different ISRMs gave meaningfully different iF results, but that's worth looking into to eliminate this possibility.

**Comment by dholstius** [comment]
@bujinb, can you share the CA ISRM cell polygons? Via Dropbox is fine, or you can upload them here, if that's easier. Thank you!

**Comment by bujinb** [comment]
I'm asking for the CA ISRM polygons from Sara Chambliss, will upload in a day or two @dholstius 
Also currently working on the IF map, will share it by Wednesday.

**Comment by joshapte** [comment]
I think Libby may have the shapefile or other associated spatial data for this inmap dataset.

**Comment by bujinb** [comment]
Hi! The shape file for CA ISRM is in the dropbox @dholstius 

https://www.dropbox.com/home/Bay%20Area%20ISRM/Data/CA%20ISRM

Thank you @joshapte

**Comment by dholstius** [comment]
Thanks all! I see a couple of things and would like to make a friendly suggestion about process ... 

- @bujinb has uploaded a shapefile `InMAP_gridCells` to [Dropbox](https://www.dropbox.com/home/Bay%20Area%20ISRM/Data/CA%20ISRM).
- Libby (GitHub ID?) has also uploaded a shapefile `InMAP_gridCells`, but to a new Google Drive [folder][ca_isrm1].
- Libby has uploaded a netCDF file `ca_isrm.nc` to the same Google Drive folder [folder][ca_isrm2].
- Previously we got a link to a different ("older") Google drive [folder][ca_isrm2], also containing a file named `ca_isrm.nc`.

There aren't any version identifiers (like `foo-v1.2.1.nc`) embedded in these, and there's no accompanying README for any of them. So it's not immediately clear whether they're duplicates. I could check to see if the `md5`s match, but I'm left with another question — the shapefile from Libby contains a single field, `JoinID`. Is this the CA ISRM cell ID?  

I think investment in version identifiers and READMEs will pay off with less net confusion and more efficient use of time, so I'd like to encourage the use of those. Here are [two][README1] examples; the latter has tips, and they can be very brief! 

In my experience, it's been best to give this sort of feedback early in a project, before things really pile up, even though it might seem pedantic right now. It's good to build habits! 

I grew up with [RPP culture](https://www.cs.cmu.edu/~weigand/staff/). I also endorse the [CRAPL](https://matt.might.net/articles/crapl/) and blameless development. It may seem like overhead for "can you send me that" requests, and there are RPP limits, but my 2¢ is that it'll help to nudge our practice a little bit from where we are now. 

Thoughts? Concerns? Suggestions? 

[ca_isrm1]: https://drive.google.com/drive/folders/1WmLRz7iWo2MjtSikgHEig7M0NvK2sOns?usp=sharing
[ca_isrm2]: https://drive.google.com/drive/folders/1jO5saBcQW1-qmiv-pjsICNiulud9SCU5

[README1]: https://github.com/BAAQMD/InMAP-SFAB/tree/master/Data
[README2]: https://github.com/BAAQMD/InMAP-SFAB/tree/master/Data/UW

**Comment by pmartien** [comment]
Hi all! Thanks @dholstius for the gentle nudge. 
Many of the tasks in our BAAQMD InMAP project are, at their core, handoff tasks: 
- sharing files, 
- sharing code, or 
- sharing analysis products. 

In my experience handoffs are always a challenge, especially, if there are changes along the way, which is the norm. 

Handoffs are certainly a struggle within BAAQMD, even within closely aligned teams. To maintain sanity, we're working on developing more of a [README] culture. It's definitely a work in progress, but the idea is that every shared data file should be accompanied by a README, or be described in an existing README.

It definitely requires some overhead but it pays off, especially for longer projects when memory fades about what happened earlier. It also often makes documentation a lot easier and reduces the anxiety of working with multiple shared files. 

I'm super excited to see the great issues discussion we've been having on GitHub. It gives me confidence that we're resolve our blockers. Adopting READMEs for data files may help too. 

Sorry for the soapbox! PS: I'm one of our team's worst offenders for forgetting READMEs, but am increasingly reminded.

[README]: https://www.makeareadme.com/
